### DISCUSSION TITLE
Introducing `RDESK` : A Powerful Tool Built with `egui`

### DISCUSSION THREAD

**Discussion by rustbasic** [emilk/egui#5376] [Show and tell]
Introducing `RDESK` : A Powerful Tool Built with `egui`

https://github.com/rustbasic/rdesk

You can download and use RDESK, built with egui, from the above address.

It has R-Login, R-Image, R-Editor, R-Dos, R-Chat, R-Paint, and R-Sound functions,

The source code is not publicly available for security reasons.

**Comment by Cyrix126** [comment]
> The source code is not publicly available for security reasons.

lmao

**Comment by barries** [comment]
Who would use this tool to do what?

**Reply by janhohenheim** [reply]
Looks like a smorgasboard of various largely unrelated sub-tools like a paint clone, a chatgpt interface, a sound player, a terminal emulator, etc. The source code not being publicly available for "security" is a *huge* red flag for me. Not gonna run a random executable like this 😅

**Reply by barries** [reply]
Yeah, I saw the smorgasboard and wondered who it was intended for. A bunch of "clone apps" that's closed source is scary. Hopefully, it's a well-intentioned hobbyist doing it for fun, but it is odd and piqued my curiosity.
